src/dotnet/Program.cs

```diff
@@ -140,6 +140,8 @@ private static int ProcessArgs(string[] args)
             {
                 // mimic the env variable
                 Environment.SetEnvironmentVariable(CommandContext.Variables.Verbose, verbose.ToString());
+                // Use the current value of AnsiPassThru but make sure child processes have it set to true.
+                CommandContext.ShouldPassAnsiCodesThrough();
```

Comment:
Is this a query or a command? If a query, should you be testing the result here? If a command, can we call it PassAnsiCodesThrough?